apps/pie-docs/src/components/modal/overview/overview.md

```diff
@@ -0,0 +1,424 @@
+
+#### Double
+
+A maximum of two Buttons are allowed within the Modal’s footer, with the primary Button right aligned and the lower emphasis Button on the left.
+
+{% contentPageImage {
+    src: "../../../assets/img/components/modal/overview/modifier-footer-double.png",
+    alt: "A modal with a close icon button in the header.",
+    width: 600
+} %}
+
+#### Single
+
+You can use the Passive modifier for the footer in instances where the user doesn't need to take an action. This will remove the buttons at the bottom of the Modal.
+
+{% contentPageImage {
+    src: "../../../assets/img/components/modal/overview/modifier-footer-single.png",
+    alt: "A modal with a pair of primary and ghost buttons in its footer.",
+    width: 600
+} %}
+
+#### Passive
+
+A single Button variant is available when only one action is required.
+
+{% contentPageImage {
+    src: "../../../assets/img/components/modal/overview/modifier-footer-passive.png",
+    alt: "A modal with a single primary button in its footer.",
+    width: 600
+} %}
+
+---
+
+#### Sizes
+
+There are three responsive Modal sizes: large, medium and small. Choose a size that works best for the amount of Modal content you have.
+
+Modals with short messages should use the small Modal to avoid long single lines; for complex or larger content, medium or large Modal will be more suitable.
+
+{% componentDetailsTable {
+  tableData: sizes,
+  priority: 2
+} %}
+
+### Large
+
+{% contentPageImage {
+    src: "../../../assets/img/components/modal/overview/size-large.png",
+    alt: "Large size modal.",
+    width: 968
+} %}
+
+### Medium
+
+{% contentPageImage {
+    src: "../../../assets/img/components/modal/overview/size-medium.png",
+    alt: "Medium size modal.",
+    width: 520
+} %}
+
+### Small
+
+{% contentPageImage {
+    src: "../../../assets/img/components/modal/overview/size-small.png",
+    alt: "Small size modal.",
+    width: 336
+} %}
+
+---
+
+## Narrow
+
+### Sizes
+
+#### Full screen
+
+The Wide large and optionally medium size, has the corresponding responsive option of fullscreen at narrow.
+
+{% contentPageImage {
+    src: "../../../assets/img/components/modal/overview/narrow-size-full-screen.png",
+    alt: "A full screen modal on a mobile screen.",
+    width: 302
+} %}
+
+#### 75% width
+
+The Wide small and optionally medium size, has the corresponding responsive option of 75% width at narrow.
+
+{% contentPageImage {
+    src: "../../../assets/img/components/modal/overview/narrow-size-75-percent.png",
+    alt: "A modal with 75% width on a mobile screen.",
+    width: 336
+} %}
+
+### Modifiers
+
+#### Footer
+
+As well as the standard Modal footer modifiers, at a narrow size you have the option to have full width Buttons that are stacked.
+
+{% contentLayout %}
+  {% contentPageImage {
+      src: "../../../assets/img/components/modal/overview/narrow-modifier-footer-dual-action.png",
+      alt: "A modal with a single primary button in its footer on a mobile screen.",
```

Comment:
Alt text should say two buttons?

Reply:
thanks

Reply:
I'd put the images backward not the alt text